### ISSUE TITLE
Shortcuts don't work right after switching background layer

### ISSUE THREAD

**Issue by Nekzuris** [openstreetmap/iD#10971]
### URL

https://www.openstreetmap.org/edit

### How to reproduce the issue?

1. Test that the `Alt + W` shortcut correctly hide the data layer
2. Change background layer
3. Without clicking anywhere else after changing background, try `Alt + W` shortcut again, it doesn't work anymore (no shortcuts work)

### Which deployed environments do you see the issue in?

Released version at openstreetmap.org/edit
Development version at ideditor.netlify.app
RapiD version at mapwith.ai/rapid

### What version numbers does this issue effect?

2.33.0

### Which browsers are you seeing this problem on?

Firefox and Chrome on Windows 10.

**Comment by tyrasd** [comment]
The cause of the issue is that while any input/form element is focused (e.g. a input text box, but also an input radio button element), keyboard shortcuts are disabled in order to allow the user to interact with the respective input element. For example to type something into a text box, or to use the arrow keys to change the selected background layer. That said, it should probably be doable to still accept "most" of the keyboard shortcuts when certain input fields are focused.
